components/button.css

```diff
@@ -64,7 +65,8 @@
     color: var(--white-color);
     background-color: var(--primary-color);
     border-radius: 50%;
-    padding: 0.8rem 1rem;
+    padding: 0.75rem 0.9rem;
```

Comment:
Suggestion: Use px for smaller values.

Reply:
Thanks for the suggestion but I prefer using rems